pipelines/datasets/br_me_rais/tasks.py

```
@@ -0,0 +1,466 @@
+    input_dir.mkdir(exist_ok=True)
+    output_dir.mkdir(exist_ok=True)
+
+    return input_dir, output_dir
+
+
+@task
+def get_source_last_year(
+    ftp_host: str = rais_constants.FTP_HOST.value,
+) -> int:
+    """Connect to MTE FTP and return the most recent available RAIS year."""
+    ftp = ftplib.FTP(ftp_host)
```

Ruff (0.15.11)

[error] FTP-related functions are being called. FTP is considered insecure. Use SSH/SFTP/SCP or some other encrypted protocol.

(S321)